private/buf/bufsync/bufsync.go

```diff
@@ -167,14 +167,13 @@ type Handler interface {
 		branch string,
 	) (git.Hash, error)
 
-	// CheckSyncedGitCommits is invoked when syncing branches to know which commits hashes from a set
-	// are already synced inthe BSR. It expects to receive the commit hashes that are synced already. If
-	// an error is returned, sync will abort.
-	CheckSyncedGitCommits(
+	// IsGitCommitSynced is invoked when syncing branches to know if a Git commit is already synced.
+	// If an error is returned, sync will abort.
+	IsGitCommitSynced(
```

Comment:
Yeah I remember when doing this I was imagining paging many git hashes in a single request, but with multi module directories that became increasingly difficult and right now we just do it one by one. I'm good with this change!